arch/arm/core/cortex_m/mpu/Kconfig

```diff
@@ -62,6 +62,30 @@ config MPU_ALLOW_FLASH_WRITE
 	help
 	  Enable this to allow MPU RWX access to flash memory
 
+config MPU_SKIP_BACKGROUND_SRAM_FILLING
```

Comment:
can we change the semantics of this Kconfig?
it would be more natural to have a "config MPU_GAP_FILLING" that is default N. And probably placed not in ARM space, since ARC also does this.

Reply:
I see the point of pushing this up in the arch KConfig; I'll do that.

Regarding the semantics; I am ok with changing the semantics; not so fond of the name. How about MPU_FULL_BACKGROUND_SRAM_PARTITIONING? (and default to N)

@andrewboie, it is important to agree on the default behavior. So you think that the default behavior on ARMv8-M (and ARC v2) should be to _not_ enforce this gap filling? Or the other way around?